docs/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md

```diff
@@ -80,6 +78,31 @@ Middlewares can have an optional configuration with the following parameters:
 | `config`  | Used to define or override the middleware configuration           | Object  |
 | `resolve` | Path to the middleware's folder (useful for external middlewares) | String  |
 
+<br/>
+
+```js
+// path: ./config/middlewares.js
+
+module.exports = [
+  // ... other internal middlewares go here
+  {
+    // adding a configuration to an internal middleware
+    name: 'strapi::cors', // notice that the name is prefixed with strapi::
+    config: {
+      origin: 'some-url.com'
+    },
+  },
+  {
+    // adding a configuration to a custom middleware 
+    resolve: 'my-custom-node-module',
+    config: {
+      foo: 'bar',
+    },
+  },
+  // ... other custom middlewares go here
```

Comment:
Why did you make a difference between internal and custom middleware? (before and after). The user chooses the order of the middleware

Reply:
Oh, sorry. Did not mean to change the order, but wanted to make sure that people know how to add a `config` to an internal middleware.
What would you suggest as an improvement?

Reply:
Honestly I've been considering having an example with each of the middlewares below their table and the options they support

Reply:
Sounds like a good idea, Derrick, but I'd suggest creating another PR for this :-)

Reply:
It's just regarding the comments line 87 and and 102. I would put the same one :
- `other middlewares go here` or
- `other middlewares (internal or custom) go here`